src/sider/arbeidsuforhet/SendForhandsvarsel84Skjema.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from "react";
+import { addWeeks } from "@/utils/datoUtils";
+import { ButtonRow } from "@/components/Layout";
+import { Button, Textarea } from "@navikt/ds-react";
+import { useForm } from "react-hook-form";
+import { SkjemaHeading } from "@/sider/aktivitetskrav/vurdering/SkjemaHeading";
+import { ForhandsvisningModal } from "@/components/ForhandsvisningModal";
+import * as Amplitude from "@/utils/amplitude";
+import { EventType } from "@/utils/amplitude";
+import { useArbeidsuforhetVarselDocument } from "@/hooks/arbeidsuforhet/useArbeidsuforhetVarselDocument";
+
+const texts = {
+  title: "Send forhåndsvarsel",
+  beskrivelseLabel: "Begrunnelse (obligatorisk)",
+  begrunnelseDescription:
+    "Begrunnelsen din blir en del av en større brevmal. Åpne forhåndsvisning for å se hele varselet.",
+  forhandsvisning: "Forhåndsvisning",
+  forhandsvisningLabel: "Forhåndsvis forhåndsvarselet",
+  missingBeskrivelse: "Vennligst angi begrunnelse",
+  sendVarselButtonText: "Send",
+  avbrytButtonText: "Avbryt",
+  malLabel: "Velg arbeidssituasjon",
+  malHelptext:
+    "Her kan du velge mellom ulike brevmaler som er tilpasset den sykmeldtes arbeidssituasjon",
+};
+
+const forhandsvarselFrist = addWeeks(new Date(), 3);
+const defaultValues = { begrunnelse: "" };
+const begrunnelseMaxLength = 1000;
+
+export interface Forhandsvarsel84SkjemaValues {
+  begrunnelse: string;
+}
+
+export const SendForhandsvarsel84Skjema = () => {
+  const {
+    register,
+    watch,
+    formState: { errors },
+    handleSubmit,
+  } = useForm<Forhandsvarsel84SkjemaValues>({ defaultValues });
+  const { getForhandsvarsel84Document } = useArbeidsuforhetVarselDocument();
+  const [showForhandsvisning, setShowForhandsvisning] = useState(false);
+
+  const submit = (values: Forhandsvarsel84SkjemaValues) => {
+    console.log(values);
+    // TODO: Mutate arbeidsuforhet (og sett loading på send-knappen)
+  };
+
+  const handlePreviewButtonClick = () => {
+    setShowForhandsvisning(true);
+    Amplitude.logEvent({
+      type: EventType.ButtonClick,
+      data: { tekst: texts.forhandsvisning, url: window.location.href },
+    });
+  };
+
+  return (
+    <form onSubmit={handleSubmit(submit)}>
+      <SkjemaHeading title={texts.title} />
+      <Textarea
+        className="mb-8"
+        {...register("begrunnelse", {
+          maxLength: begrunnelseMaxLength,
+          required: true,
```

Comment:
Det går evt an å si `required: texts.missingBeskrivelse` her (dvs påkrevd med den feilmeldingen). Da kan man forenkle til `error={errors.begrunnelse?.message}` under (som vil vise den feilmeldingen vi definerte for `required` over)

Reply:
Fancy! 😮